python/lsst/ip/isr/isrFunctions.py

```diff
@@ -797,6 +815,10 @@ def gainContext(exp, image, apply):
             ccd = exp.getDetector()
             for amp in ccd:
                 sim = image.Factory(image, amp.getBBox())
+                if gains:
+                    gain = gains[amp.getName()]
+                else:
+                    gain = amp.getGain()
                 sim /= amp.getGain()
```

Comment:
`sim /= gain`

Reply:
Awesome 👏 _This_ is what reviews are all about. That would have been a nightmare to find if you hadn't caught it!